src/main/kotlin/com/cognifide/gradle/aem/environment/docker/domain/HttpdContainer.kt

```diff
@@ -10,11 +10,11 @@ class HttpdContainer(environment: Environment) {
 
     private val aem = environment.aem
 
-    val container = DockerContainer("aem_httpd")
+    val container = DockerContainer(aem, aem.props.string("environment.httpdContainer.containerName") ?: "aem_httpd")
```

Comment:
I would use `gap` prefixes since `aem` is too wide. GAP is specific and clashes are less feasible.

Reply:
No matter 😉